packages/router/router.coffee

```diff
@@ -39,3 +39,22 @@ FlowRouter.route '/organizations/:_id',
     FlowLayout.render 'layout',
       main: 'organizationDetail'
       params: {"organizationId": params._id}
+
+FlowRouter.route '/myDatasets',
```

Comment:
Maybe @jgoley could do a mockup. My intuition is that would be too much to fit comfortably on a single page.